### PR TITLE
Updated BB Section

### PR DESCRIPTION
**Pull request by the-real-kenna**
I've amended / added the following info: 
* updated the kernel version BB runs on
* added warning that capemgr is no longer supported in current versions of balenaOS
* added instructions for adding an overlay (which replaces capemgr)

Change-type: patch
Signed-off-by: Kenna Smith <kenna@balena.io>

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

File: pages/learn/develop/hardware/i2c-and-spi.md [hunk 1]
```diff
@@ -209,15 +209,15 @@ overscan_bottom=4
 
 ## Beaglebone
 
-Currently the Beaglebone devices are running a very new 4.1 kernel (which is obviously awesome), unfortunately many of the userspace libraries haven't caught up yet so they only work with the older 3.8 kernel. Luckily [ruth0000](https://github.com/ruth0000) was kind enough to patch the Octalbonescript JS library and made a lovely node.js module over here: https://www.npmjs.com/package/octalbonescript_capemgr4_1 .
+Currently the Beaglebone devices are running a v5.4 kernel (as of March 2021), unfortunately many of the userspace libraries haven't caught up yet so they only work with the older 3.8 kernel. Luckily [ruth0000](https://github.com/ruth0000) was kind enough to patch the Octalbonescript JS library and made a lovely node.js module over here: https://www.npmjs.com/package/octalbonescript_capemgr4_1 .
```
Comment (vipulgupta2048) [inline pages/learn/develop/hardware/i2c-and-spi.md:212]: @kenna-smith I would recommend not adding dates in documentation as it leads to redundancy much faster. Instead how about. 

`Currently, the Beaglebone devices are running a v5.4 kernel ([Check latest kernel version here](LINK TO KERNEL))`

**Review by vipulgupta2048 — COMMENTED** [review]
LGTM, do self-certify when ready. 
Reference: https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md#balenistas-only-using-versionbot-to-manage-your-pull-request

**Comment by vipulgupta2048** [comment]
@kenna-smith pinging again

File: pages/learn/develop/hardware/i2c-and-spi.md [hunk 2]
```diff
@@ -234,6 +234,14 @@ cat /sys/devices/platform/ocp/ocp:cape-universal/status
 OCPDIR=/sys/devices/platform/ocp/ocp*
 SLOTS=/sys/devices/platform/bone_capemgr/slots
 ```
+### Adding an Overlay
+
+To load an overlay in more recent versions of balenaOS, it's necessary to edit the boot partition uEnv.txt file (or uEnv.txt_internal on the flasher image) before provisioning and add the following content:
```
Comment (floion) [inline pages/learn/develop/hardware/i2c-and-spi.md:239]: @kenna-smith if you write here provisioning it's implied that it uses a flasher image. So the reference would be actually only to uEnv.txt_internal

**Pull request closed without merging**